packages/solid-crs-manage/package.json

```diff
@@ -23,7 +23,7 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "lint:staged": "lint-staged",
-    "test": "jest",
+    "test": "node --trace-warnings node_modules/.bin/jest",
```

Comment:
was this helpful in some way? otherwise remove

Reply:
This helps in finding which promises aren't handled correctly ( either by a test or by the tested object ).

Reply:
is calling jest with node from node_modules needed for that as well?

Reply:
the `--trace-warnings` is only a parameter for `node` and as far as I know jest does not have a similar parameter.
Not sure if this is THE way, but this is what was suggested on various stackoverflows and other similar sites.

I'ill keep this conversation open and revert it at the end of the PR.

Reply:
created https://github.com/netwerk-digitaal-erfgoed/solid-crs/issues/566 to investigate other unhandled promise rejections using the `node --trace-warnings node_modules/.bin/jest` command